src/parser/shadowlands/modules/items/dungeons/CodexOfTheFirstTechnique.tsx

```diff
@@ -0,0 +1,85 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { AbsorbedEvent, DamageEvent } from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import Buffs from 'parser/core/modules/Buffs';
+import BoringItemValueText from 'parser/ui/BoringItemValueText';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+class CodexOfTheFirstTechnique extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+    buffs: Buffs,
+  };
+
+  protected buffs!: Buffs;
+  protected damage = 0;
+  protected healing = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTrinket(ITEMS.CODEX_OF_THE_FIRST_TECHNIQUE.id);
+
+    if (!this.active) {
+      return;
+    }
+
+    (options.buffs as Buffs).add({
+      spellId: SPELLS.CODEX_OF_THE_FIRST_TECHNIQUE_HEALING.id,
+      timelineHighlight: true,
+    });
+
+    (options.abilities as Abilities).add({
+      spell: SPELLS.CODEX_OF_THE_FIRST_TECHNIQUE_HEALING.id,
+      category: Abilities.SPELL_CATEGORIES.ITEMS,
+      gcd: null,
+    });
```

Comment:
this *might* add the trinket to the abilities list on the stats page, but i'm not sure. gonna merge and find out bc its such a low-impact error if that is the case